hooks/queries/votes/useRevealedVotes.ts

```diff
@@ -19,7 +21,25 @@ export function useRevealedVotes(addressOverride?: string) {
 
   const queryResult = useQuery(
```

Comment:
makes sense!

this will achieve the same result in a bit of a cleaner way by using the built-in `onSuccess` hook

```suggestion
    const queryResult = useQuery({
    queryKey,
    queryFn: () => getRevealedVotes(voting, address, roundId),
    onSuccess(data) {
      queryClient.setQueryData<typeof data>(queryKey, (oldRevealedVotes) =>
        oldRevealedVotes ? { ...oldRevealedVotes, ...data } : data
      );
    },
    onError,
    initialData: {},
    enabled: !!address && !isWrongChain,
  });
```

Reply:
nice, will change this